refactor(formatter): Remove unused multiline comment check in ternary

leaysgur opened the pull request:
Follow up #16224 

> - Biome
>   - The same logic still exists
>     - https://github.com/biomejs/biome/blob/e7e0f6c14df92d83d08f86b1e57fc82b4df775b7/crates/biome_js_formatter/src/utils/conditional.rs#L197
>   - In either case, the test did not fail
>     - always returning `false`
>     - removing related code like our diff above
> - Prettier
>   - The most relevant lines are here
>     - https://github.com/prettier/prettier/blob/fd69335e513232c9b7aad78c93b42d8d3dd00ae7/src/language-js/print/ternary-old.js#L326-L344
>   - As you expected, if change this to `const shouldBreak = false`, the test did not fail!

From https://github.com/oxc-project/oxc/pull/16224#discussion_r2570513632

Dunqing commented:
### Merge activity

* **Nov 28, 7:12 AM UTC**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.com/merges?org=oxc-project&repo=oxc) once it meets the requirements.
* **Nov 28, 7:12 AM UTC**: `Dunqing` added this pull request to the [Graphite merge queue](https://app.graphite.com/merges?org=oxc-project&repo=oxc).

leaysgur committed:
refactor(formatter): Remove unused multiline comment check in ternary (#16230)

Follow up #16224

> - Biome
>   - The same logic still exists
>     - https://github.com/biomejs/biome/blob/e7e0f6c14df92d83d08f86b1e57fc82b4df775b7/crates/biome_js_formatter/src/utils/conditional.rs#L197
>   - In either case, the test did not fail
>     - always returning `false`
>     - removing related code like our diff above
> - Prettier
>   - The most relevant lines are here
>     - https://github.com/prettier/prettier/blob/fd69335e513232c9b7aad78c93b42d8d3dd00ae7/src/language-js/print/ternary-old.js#L326-L344
>   - As you expected, if change this to `const shouldBreak = false`, the test did not fail!

From https://github.com/oxc-project/oxc/pull/16224#discussion_r2570513632
